.github/scripts/trymerge.py

```diff
@@ -52,6 +57,17 @@ def __init__(self, name: str, url: str, status: Optional[str]):
         self.status: Optional[str] = status
         self.jobs: JobNameToStateDict = {}
 
+class FlakyRule:
+    def __init__(self, name: str, captures: List[str]):
+        self.name: str = name
```

Comment:
Why type annotation is needed there?
```suggestion
        self.name = name
```

Reply:
removed